cdap-proto/src/main/java/co/cask/cdap/proto/metadata/MetadataRecord.java

```diff
@@ -25,7 +25,7 @@
 import java.util.Set;
 
 /**
- * Represents the complete metadata of a {@link Id.NamespacedId} including its properties and tags.
+ * Represents the complete metadata of a {@link Id.NamespacedId} including its properties, tags and scope.
```

Comment:
the metadata of an entity does not include its scope. Maybe "Represents the complete metadata of a {@link Id.NamespacedId} including its properties and tags in a given {@link MetadataScope}"